### ISSUE TITLE
Changelog for pre 2.0?

### ISSUE THREAD

**Issue by mwarkentin** [SumoLogic/terraform-provider-sumologic#189]
Hello,

We have an old sumologic provider (0.2.1) that we're looking at upgrading. It'd be helpful to have a changelog for the older versions so that we can see what we're getting ourselves into ahead of time. :)

**Comment by seansain** [comment]
I found these Git Logs for our old releases. Hope this helps.

v1.3.0
```
15b1813 Add collector import by name functionality
e188690 Add collectorName/sourceName import logic
260fdde Add to provider file
f820fd5 Don't recreate source on name/fields change
2154b9d Fix all errors
049053b Fix error cases
85f8987 Fix file name and url
fb13db2 Fix names
11b0ad8 Fix source import for missing collector/source
9b42de4 Fixes
90159bf Fixes #80. Adding missing /v1 in the URL
240b8c1 Fixes token bug in cloudsyslog_source resource
5c95dbf Merge branch 'master' into bug/tf12-build
31bbeb9 Merge pull request #63 from spothero/bug/tf12-build
270c561 Merge pull request #76 from SumoLogic/rmiller-add-collector-import-by-name
199a864 Merge pull request #77 from SumoLogic/rmiller-disable-force-new-source-fields
ec51063 Merge pull request #79 from SumoLogic/ggarg-partition
609bdcd Merge pull request #81 from SumoLogic/mccartney-v1-for-scheduled-views
96d739a Merge pull request #82 from SumoLogic/rmiller-fix-tf-source-import
f208158 Partition Support
c0e9e2f Update README with new import commands
9c26b24 Update sumologic_scheduled_view.md
6e56c8a Use new get collector by name API
```

v1.2.0
```15b1813 Add collector import by name functionality
e188690 Add collectorName/sourceName import logic
f820fd5 Don't recreate source on name/fields change
049053b Fix error cases
240b8c1 Fixes token bug in cloudsyslog_source resource
5c95dbf Merge branch 'master' into bug/tf12-build
31bbeb9 Merge pull request #63 from spothero/bug/tf12-build
270c561 Merge pull request #76 from SumoLogic/rmiller-add-collector-import-by-name
199a864 Merge pull request #77 from SumoLogic/rmiller-disable-force-new-source-fields
c0e9e2f Update README with new import commands
9c26b24 Update sumologic_scheduled_view.md
6e56c8a Use new get collector by name API
```

v1.1.0
```
6cef85a Add fields
bcdd649 Adding docs for scheduled_view
6a1ad2b Adding folder management, also updating API request URLs as content/folders are V2, not V1
5db8b48 Adding info on how to find the parent id
28e2034 Adding support for specifying baseURL explicitly
2acf48d Avoid copy&paste, refer to the docs/
0677abd Code reformatted with gofmt
53e13f1 Create sumologic_folder.md
957e79e Fixed links
275ef17 Fixes
d913291 Fixing docs link
5067623 Fixing the import warning
b18ffac Implemented Read, also dealing with RetentionPeriod defaults
653ba05 Implemented Update and Delete
6aa4790 Indent fixes
beef8f4 Merge branch 'master' into mccartney-baseurl
6843f21 Merge branch 'master' into mccartney-baseurl
862f3ae Merge pull request #61 from mtanda/roles_doc
4f3c535 Merge pull request #62 from mpandya-sumo-logic/master
af71c76 Merge pull request #65 from SumoLogic/mccartney-import-case
e14ef98 Merge pull request #66 from SumoLogic/mccartney-clean-up-docs
2e223f8 Merge pull request #67 from SumoLogic/mccartney-scheduled-views
9f98ad3 Merge pull request #70 from SumoLogic/mccartney-baseurl
28c6fe4 Merge pull request #71 from SumoLogic/mccartney-fixing-apache-thrift
01a4a85 Merge pull request #73 from SumoLogic/mccartney-docs-for-scheduled-views
cc9130f Merge pull request #75 from SumoLogic/add-fields-support
56b1fe5 Restructure, first the navigation links, only then the common sources properties
2c8332b Scheduled Views creation
72b9a18 StartTime type handling
968b2aa Update resource_sumologic_folder.go
a196207 Update sumologic_folder.md
939621c Updated the documentation
f1f613b Updating indentation
661506d Updating schema with minor change (changing a required parameter in a computed field to optional)
b0ca7c8 Updating struct alignment
f5a0e52 Updating sumologic capitalization to resolve issues during building
4978678 Upgrading to Go 1.13 to fix Apache git issue
3bb0bbf add doc for role resource
4ec1f57 link to doc fixed
```

v1.0.0
```
5955b44 Merge pull request #54 from hamstah/terraform-0.12
1ab3d24 Upgrade dependencies for terraform 0.12
875cb7a Use go 1.12
504e3a3 set GO111MODULE for travis
```

v0.4.0
```
4caedd7 Add Argument reference to doc
ae3bc6b Add Attributes reference
a682d77 Add example
209e251 Add initial Doc for users
e517731 Add isActive to Put
3e0761d Add support for ingest budgets along with assigning collectors to them
6c12943 Add validation
a32c313 Correct JSON tag for cloud syslog token
ed22ca8 Correct the repeat of firstName
51d4a38 Corrections to the Put actions
e16a017 Cosolidate to one struct using omitempty
5777392 Improve resourceSumologicUserExists
c926cf4 Merge branch 'master' into user
b51535f Merge branch 'user' of https://github.com/heldersepu/sumologic-terraform-provider into user
16e2808 Merge conflict
137ea43 Merge pull request #48 from spothero/correct-json-token
3ffd0e5 Merge pull request #49 from bendrucker/docs-http-source-example
3a5a75d Merge pull request #51 from heldersepu/user
08709ab Merge pull request #57 from wjam/ingest-budget
78a8685 Merge pull request #59 from heldersepu/client
26fb538 Model User: https://api.sumologic.com/docs/#operation/updateUser
b0899dd Remove destroy
33dcbe5 Remove testing changes
fd1953f Revert changes on imports
15d994c Simplify error handling
341bc9e Update sumologic_polling_source.md
76b8b28 User API support
33e34ca indentation
```

v0.3.0
```
eaa23ad Add filter example
0de5059 Add note about SUMOLOGIC_ENVIRONMENT variable
9ee3abe Merge pull request #37 from jolexa/patch-1
ccfa6f3 Merge pull request #39 from zioproto/filter-example
d1bcfd7 Merge pull request #40 from bendrucker/http-source-import
93a103c Merge pull request #42 from mtanda/role
8873cf7 Merge pull request #43 from mtanda/fix_role
e5dadcf Merge pull request #46 from bendrucker/docs-http-import-typo
c6e365a Update README.md
085941e Update default environment docs
2f10338 Update sumologic_polling_source.md
f58f64b add Japan endpoint
327d0a7 add docs for http_source import
686e3c5 add go import path
b6ba06b add import support for http sources
21eec53 add test
9ac0453 fix #38 - log request body in event of failure
52b6e66 fix bug
1146850 fix random order issue of string list
5d58e57 fix validation error
0ec7ffc remove accessKey from caller identity
f262ceb remove accessKey from caller identity
81a78c6 remove sumologic TF code from vendor folder, add new Canada deployment
b8dbd6f role support
6f36a86 validate input
```

**Comment by mwarkentin** [comment]
That's great, thanks!
